sites/partners/src/components/listings/PaperListingForm/index.tsx

```diff
@@ -247,6 +267,14 @@ const ListingForm = ({ listing, editMode, setListingName }: ListingFormProps) =>
           clearErrors()
           const successful = await formMethods.trigger()
 
+          if (whatToExpectEditor?.storage.characterCount.characters() > CHARACTER_LIMIT) {
+            setLoading(false)
+            setAlert("form")
+            return
+          }
+
+          formData.whatToExpect = cleanRichText(whatToExpectEditor.getHTML())
```

Comment:
issue: @emilyjablonski this line causes issue if you do the following steps:
1. Go to add a listing
2. Click Publish
3. Confirm publish on modal
4. The "something went wrong" error appears and it never actually called the backend.
This is because `whatToExpectEditor` is null. We could change it to `whatToExpectEditor?.getHTML()` however I think the better solution is to add the `whatToExpectEditor` to the list of dependencies for this `useCallback` function. Do you see any issue with that? Also, I'm curious how this is working in core but not here.